index.html

```diff
@@ -0,0 +1,250 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Worrior' Cars</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="headerDesktop">
+      <div class="links">
+        <ul>
+          <li> <a href="https://web.facebook.com/?_rdc=1&_rdr"><img class="facebook-icon" src="Icons/facebook.png" alt="Facebook Icon" /></a> </li>
+          <li> <a href="https://twitter.com/home"><img class="twitter-icon" src="Icons/twitter.png" alt="Twitter Icon" /></a></li>
+          <li>English</li>
+          <li>MyPage</li>
+          <li>Logout</li>
+        </ul>
+      </div>
+      <div class="headerSecondary">
+          <nav>
+            <div>
+               <img id="mobileMenu" src="Icons/mobile-menu.png" alt="Mobile-Menu Icon"> 
+               <img id="Mobilecross" src="Icons/cross.png" alt="Cross Icon">
+            </div>
+          <span class="menu">
+           <a href=""><img src="icons/warrior.png" alt="Warrior Icon" /></a>
+            <ul>
+              <li id="about-item" class="about-item" >About</li>
+              <li>Program</li>
+              <li>Join</li>
+              <li>Sponsor</li>
+              <li>News</li>
+            </ul>
+            <button>Compaign</button>
+          </span>
+        </nav>
+      </div>
+    </header>
+
+    <div id="PopupMobileMenu">
+      <ul>
+        <div id="X">X</div>
+        <li id="about-item-mob" class="about-item" >About</li>
+        <li>Program</li>
+        <li>Join</li>
+        <li>Sponsor</li>
+        <li>News</li>
+      </ul>
+    </div>
+
+<main class="desktop-main" id="desktop-main">
+    
+    <section class="landing-page">
+        <div>
+            <h3>"The New Release!"</h3>
+            <h1>
+                Announcing Our <br />
+                Jaguare E-3
+                2023
+            </h1>
+            <p>
+                Worrior has released its latest electric car, the e-type 3.This
+                vehicle boasts a sleek design,improved driving performance, and a
+                longer battery range.
+            </p>
+            <h2 id="landing-pageH2">2023.2.7(MON) ~ 12(SAT)</h2>
+            <h4 id="landing-pageH4">@ F-9 Park, Panjab Gate, Islamabad</h4>
```

Comment:
- [ ] As suggested by the previous reviewer please make sure to use `<hx>` tag hirarecy properly, in a way that `<h1>` tag should come first then `<h2>`,  `<h3>` and so on ...

The purpose of the `heading` tags is semantics. Correct use of headings on a page provides hierarchy and overview, and helps quickly communicate what a web page is about. They help visitors quickly scan a page, and they help search engines quickly understand a page's structure and what it's about. If you want your heading to be a particular font please use `CSS` properties for that.